streams/src/main/scala/tofu/streams/Broadcast.scala

```diff
@@ -0,0 +1,12 @@
+package tofu.streams
+
+trait Broadcast[F[_]] {
```

Comment:
Is this some kind of optimization abstraction?

Reply:
@Odomontois, right, I think this functionality is too specific.

Reply:
I don't mind it while you are going to support it